content/docs/develop/installation/upgrade-keptn/index.md

```diff
@@ -6,84 +6,22 @@ icon: setup
 keywords: upgrade
 ---
 
-For upgrading an existing keptn 0.2.0, 0.2.1 or 0.2.2 installation, an upgrade script is provided. This will update all keptn core components to their new version and installs the keptn's bridge.
+Since the removal of knative as a core technology in keptn 0.4.0 is considered to be a breaking change, an upgrade script can unfortunately not be provided. If you want to upgrade your existing keptn installation, uninstall keptn and install it with the new keptn CLI 0.4.0.
 
-## Prerequisites
-   
-- Please note that all [command line tools](../setup-keptn-gke#prerequisites) are needed when upgrading keptn.
-  
-    - Additionally, [yq](https://github.com/mikefarah/yq) is required.
+## Uninstall keptn
 
-- Furthermore, please note that we increaded the GKE cluster size to one `n1-standard-16` node.
+For uninstalling keptn, [download this script](https://github.com/keptn/installer/blob/0.4.0/scripts/common/uninstallKeptn030.sh) and execute it:
```

Comment:
Are we uninstalling Dynatrace or any other components that not belong to keptn core during that process?

Reply:
No it does only uninstall keptn core.